webview-ui/src/components/settings/sections/FeatureSettingsSection.tsx

```diff
@@ -169,6 +172,63 @@ const FeatureSettingsSection = ({ renderSectionHeader }: FeatureSettingsSectionP
 							</p>
 						</div>
 					)}
+					{isDictationFeatureEnabled && (
+						<>
+							<div style={{ marginTop: 10 }}>
+								<VSCodeCheckbox
+									checked={dictationSettings?.dictationEnabled}
+									onChange={(e: any) => {
+										const checked = e.target.checked === true
+										const updatedDictationSettings = {
+											...dictationSettings,
+											dictationEnabled: checked,
+										}
+										updateSetting("dictationSettings", updatedDictationSettings)
+									}}>
+									Enable Dictation
+								</VSCodeCheckbox>
+								<p className="text-xs text-[var(--vscode-descriptionForeground)] mt-1">
+									Enables speech-to-text transcription using your Cline account. Uses the Whisper model, at
+									$0.006 credits per minute of audio processed. 5 minutes max per message.
+								</p>
+							</div>
+
+							{/* TODO: Fix and use CollapsibleContent, the animation is good but it breaks the dropdown
+							<CollapsibleContent isOpen={dictationSettings?.dictationEnabled}> */}
+							{dictationSettings?.dictationEnabled && (
+								<div style={{ marginTop: 10, marginLeft: 20 }}>
+									<label
+										className="block text-sm font-medium text-[var(--vscode-foreground)] mb-1"
```

Comment:
You can check out webview-ui/tailwind.config.mjs to see what color is available as tailwind 
```suggestion
										className="block text-sm font-medium text-foreground mb-1"
```